aptos-move/framework/aptos-token/doc/token.md

```diff
@@ -138,13 +138,15 @@ Checkout our developer doc on our token standard https://aptos.dev/concepts/coin
 <code>amount: u64</code>
 </dt>
 <dd>
-
+ the amount of tokens. Only property_version = 0 can have a value bigger than 1.
```

Comment:
Why we can an update on this auto-generated file?

Reply:
Good call! This change has been removed from this commit when I rebased it onto the head.